scripts/migrate-to-bcd.ts

```diff
@@ -0,0 +1,90 @@
+import assert from 'node:assert';
+import fs from 'node:fs';
+import path from 'node:path';
+import {fileURLToPath} from "node:url";
+import { fdir } from 'fdir';
+import { features } from '../index.js';
+
+export const BCD_PATH = process.env.BCD_PATH
+  ? path.resolve(process.env.BCD_PATH)
+  : fileURLToPath(new URL("../../browser-compat-data", import.meta.url));
```

Comment:
The mdn-bcd-collector uses an environment variable, so I thought it would be best to use an environment variable for consistency with other tooling.  If you'd really like me to use `yargs`, I can, although since the goal is to run this script automatically using GitHub Actions, I don't think it will make much of a difference.

Reply:
Yes, I would. It's mostly about about help/documentation and the principle of least surprise. Env vars as the sole method of runtime configuration is unusual for web-features, even if it's not for other projects. This isn't true for everything, but I'd expect something like `npm run migrate-to-bcd -- --help` or `npm run migrate-to-bcd -- --bcd-dir=…` to do something useful.

(I guess now's a good time to add this: I think having a default directory is quite surprising too. It didn't _work_ but it also didn't _fail_ the first time I ran it because I don't have BCD checked out at the default path. I think requiring a path would be much less surprising and doing so with yargs would get us help and error messages too.)

Also, using env vars is still possible, so it doesn't conflict with the collector's conventions. You can set yargs options with env vars, if you set a prefix (which is also a good practice): https://yargs.js.org/docs/#api-reference-envprefix. That would be quite friendly to GitHub Actions and tools like `direnv`.

Reply:
@queengooborg do you plan to come back to this?